actfw_core/unicam_isp_capture.py

```diff
@@ -146,14 +153,20 @@ def __init__(
             # Auto selection of unicam_size and crop_size.
             if self.sensor_name == "imx219":
                 self.auto_size_selection_for_imx219()
+            elif self.sensor_name == "imx708":
+                self.auto_size_selection_for_imx708()
             else:
                 self.auto_size_selection_for_ov5647()
         else:
             raise RuntimeError("Both unicam_size and crop_size must be None or tuples.")
 
         # control values
+        if self.sensor_name == "imx708":
+            config_file = self.get_subdevice_name(unicam_subdev) + ".json"
+        else:
+            config_file = self.sensor_name + ".json"
         # config precedence: default < sensor specific < user given
-        with open(path.join(path.dirname(__file__), "data", self.sensor_name + ".json"), "r") as f:
+        with open(path.join(path.dirname(__file__), "data", config_file), "r") as f:
```

Comment:
Nits: I think it’s better to explicitly throw an error when the sub-device name is anything other than ‘imx708_wide’.